documentation/doc.css

```diff
@@ -119,6 +119,11 @@ html {
     margin: 1rem 2rem;
 }
 
+/* nav-bar component */
+.margin-nav{
+    margin: 2rem;
+}
```

Comment:
Also, the margin-nav is general class and doesn't tell us about the exact margin. You may give utility classes for that give margins.

Reply:
ok i will do